starter-code/iteration1/src/app/app.component.ts

```diff
@@ -6,5 +6,10 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app works!';
+  image : string ='https://i.pinimg.com/736x/43/b0/fa/43b0fa638658f271de1e9465b3c56afa--cool-cats-hippie-boho.jpg';
+  imageDescription: string="Nyan Cat cat"
+  image2 : string = 'https://i.imgur.com/3R3iYvP.jpg';
+  imageDescription2: string="Galaxy cat"
+  image3 : string = 'https://i.ytimg.com/vi/ucj9M-YXulY/hqdefault.jpg'
+  imageDescription3: string="Cat cat"
 }
```

Comment:
We do see some kind of repetition here: each image has source (url) and some description. The proper way would be to make the array of images. You have to understand basic concepts of Angular, please ask us for help if you still feel insecure about this.